### PR TITLE
Remove the word subforum everywhere it appears

### PR DESCRIPTION
**Pull request by Will-Howard**
In this PR:
 - Replaces the word "subforum" with "(core) topic" wherever it appears in the UI
 - Make the left sidebar topic list be a list of core topics rather than subforum
 - Change the new posts page tag selector UI
 - Change the plain topic link to go to the wiki tab on subforum pages
 - I haven't tried too hard to change the word subforum in the code, because it is quite deeply embedded (it appears like 800 times)

Screenshot 2023-02-03 at 11 26 41
Screenshot 2023-02-03 at 11 28 15
Screenshot 2023-02-03 at 11 28 53
Screenshot 2023-02-03 at 11 29 13

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203892226812238) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/common/TabNavigationMenu/SubforumsList.tsx [hunk 1]
```diff
@@ -38,11 +39,13 @@ const INITIAL_LIMIT = 3
 
 const SubforumsList = ({ onClick, classes }) => {
   const { results } = useMulti({
-    terms: {view: 'currentUserSubforums'},
+    // TODO possibly make it include subscribed tags
+    terms: {view: 'coreTags'},
```
Comment (Will-Howard) [inline packages/lesswrong/components/common/TabNavigationMenu/SubforumsList.tsx:43]: This is the thing I'm least sure of. I think people might be annoyed that they have joined subforums which now won't appear in the sidebar, but I don't want to do too heavy a redesign at this point

File: packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx [hunk 2]
```diff
@@ -252,7 +252,7 @@ export const menuTabs: ForumOptions<Array<MenuTab>> = {
       showOnCompressed: true,
     }, {
       id: taggingNamePluralSetting.get(),
-      title: taggingNamePluralCapitalSetting.get() + " Wiki",
```
Comment (Will-Howard) [inline packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx:255]: Also change "Topics Wiki" to "Topics", which arguably doesn't make sense in this PR but we were going to do it anyway

File: packages/lesswrong/components/tagging/subforums/SubforumSubforumTab.tsx [hunk 3]
```diff
@@ -144,17 +142,14 @@ const SubforumSubforumTab = ({tag, userTagRel, layout, isSubscribed, classes}: {
...
   const discussionButton = (
     <LWTooltip
-      title={
-        canPostDiscussion
-          ? "Create a discussion which will only appear in this subforum"
-          : "You must be a member of this subforum to create a discussion"
-      }
+      title={"You must be logged in to create a discussion"}
+      // TODO merge with JP's PR and just disable for logged in users
+      // disabled={currentUser}
```
Comment (Will-Howard) [inline packages/lesswrong/components/tagging/subforums/SubforumSubforumTab.tsx:149]: Need to change this before merging

File: packages/lesswrong/lib/collections/tags/helpers.ts [hunk 4]
```diff
@@ -21,19 +21,20 @@ export const tagMinimumKarmaPermissions = forumSelect({
...
+  tab?: string
 }
 
 export const tagUrlBase = taggingNameIsSet.get() ? taggingNamePluralSetting.get() : 'tag'
 export const tagCreateUrl = `/${tagUrlBase}/create`
 export const tagGradingSchemeUrl = `/${tagUrlBase}/tag-grading-scheme`
 
-export const tagGetUrl = (tag: {slug: string, isSubforum: boolean}, urlOptions?: GetUrlOptions) => {
-  // Assume links that are not explicitly for the subforum should go to the wiki tab (this may change in the future)
-  const urlSearchParams = tag.isSubforum ? {tab: "wiki", ...urlOptions} : urlOptions
```
Comment (Will-Howard) [inline packages/lesswrong/lib/collections/tags/helpers.ts:32]: This function already appended `?tab=wiki` to the url, so just removing this and making the plain url default to the wiki should keep all the links the same

**Review by s-cheng — APPROVED** [review]
Nice thanks! Just found a couple small things

**Pull request merged**